bundles/org.openhab.binding.myuplink/README.md

```diff
@@ -0,0 +1,79 @@
+# myUplink Binding
+
+The myUplink binding is used to get "live data" from from Nibe heat pumps without plugging any custom devices into your heat pump.
+This avoids the risk of losing your warranty.
+Instead data is retrieved from myUplink.
+The myUplink API is the successor of the Nibe Uplink API.
+This binding should in general be compatible with all heat pump models that support myUplink.
+Read or write access is supported by all channels as exposed by the API.
+Write access might only be available with a paid subscription for myUplink.
+You will need to create credentials at https://dev.myuplink.com/apps in order to use this binding.
+
+## Supported Things
+
+This binding provides two thing types:
+
+| Thing/Bridge        | Thing Type          | Description                                                                                   |
+|---------------------|---------------------|-----------------------------------------------------------------------------------------------|
+| bridge              | account             | cloud connection to a myUplink user account                                                   |
+| thing               | genericDevice       | the physical heatpump which is connected to myUplink                                          |

```

Comment:
Please adjust Thing id to the naming convention `lower-case-hypen`
For reference: https://www.openhab.org/docs/developer/guidelines.html#naming-convention

Reply:
fixed it